api/envoy/config/filter/http/grpc_stats/v2alpha/config.proto

```diff
@@ -53,6 +53,11 @@ message FilterConfig {
     //   `envoy.deprecated_features.grpc_stats_filter_enable_stats_for_all_methods_by_default`.
     google.protobuf.BoolValue stats_for_all_methods = 3;
   }
+
+  // If true, the filter will gather a histogram for the request time of the upstream.
+  // It works with stats_for_all_methods and individual_method_stats_allowlist the same way
+  // request_message_count and response_message_count works.
+  bool enable_upstream_stats = 4;
```

Comment:
v2 API is now frozen. Please revert this, and only change the v3 API.

Reply:
Should I change the doc to point to v3 then also?